tools/verify_wheels.py

```diff
@@ -438,36 +439,38 @@ def model_and_pi_schedule():
     return model, schedule
 
 if __name__ == '__main__':
-    # Run qasm simulator
+    # Run Aer simulator
     shots = 4000
     circuits = grovers_circuit(final_measure=True, allow_sampling=True)
     targets = [{'0x0': 5 * shots / 8, '0x1': shots / 8,
                 '0x2': shots / 8, '0x3': shots / 8}]
-    simulator = QasmSimulator()
-    qobj = assemble(transpile(circuits, simulator), simulator, shots=shots)
-    result = simulator.run(qobj).result()
-    print(result.status)
-    import pprint
-    pprint.pprint(result.to_dict())
+    simulator = AerSimulator()
+    result = simulator.run(transpile(circuits, simulator), shots=shots).result()
     assert result.status == 'COMPLETED'
+    assert result.success is True
```

Comment:
With the increased duplication here, it's starting to feel like a separate function might be a bit more readable, but that's not a big deal at all.